### ISSUE TITLE
InputBar overlaps messages

### ISSUE THREAD

**Issue by llKoull** [MessageKit/MessageKit#828]
Hello!

I'm having troubles with the inputBar, it always overlaps the messages :S

I've got a UIViewController in my Storyboard, on that view controller I have a view with some information and a containerView with the messages.

![storyboard](https://user-images.githubusercontent.com/5525265/44617741-c03ba680-a868-11e8-96cd-bce4cc5788f8.jpeg)

As you can see the bottom constraint of the container view is aligned with the bottom of the safe area (i also tried aligning it with the bottom of the view) and it seems to work nicely:

![simulator screen shot - iphone 6 - 2018-08-25 at 13 15 51](https://user-images.githubusercontent.com/5525265/44617755-0b55b980-a869-11e8-8a90-8fbaa4ed51ea.png)

But the problem is that when i call self.becomeFirstResponder() method from the viewWillAppear of the controller of the containerView the messages gets overlapped:

![simulator screen shot - iphone 6 - 2018-08-25 at 13 17 44](https://user-images.githubusercontent.com/5525265/44617772-4bb53780-a869-11e8-8840-85eaff838578.png)

It occurs in all iPhone sizes (including iPhone X) and i tried to change some settings like the opacity of the inputBar and disabling the option "under opaque bars" and "under bottom bars" of the storyboard but nothing seems to work. I also tried to set the first responder in the viweDidLayout() method but it did work either.

I've been searching the other issues looking for an answer and i tried some things that didn't work.

Can you help me??

This is my initialization in the ViewWillAppear() method of the containerView controller right now:

```
override func viewWillAppear(_ animated: Bool)
    {
        super.viewWillAppear(animated)
                
        self.pusherChatKitManager.getMessagesForRoom(self.chatRoom, completion: { messages in
            DispatchQueue.main.async {
                self.messages = messages.reversed()
                self.messagesCollectionView.reloadData()
                self.messagesCollectionView.scrollToBottom()
                
                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
                    self.becomeFirstResponder()
                }
            }
        })
        
        self.messagesCollectionView.messagesDataSource = self
        self.messagesCollectionView.messagesLayoutDelegate = self
        self.messagesCollectionView.messagesDisplayDelegate = self
        self.messagesCollectionView.messageCellDelegate = self
        self.messageInputBar.delegate = self
        
        self.messageInputBar.sendButton.tintColor = #colorLiteral(red: 0, green: 0.7333333333, blue: 0.7137254902, alpha: 1)
        self.scrollsToBottomOnKeybordBeginsEditing = true
        self.maintainPositionOnKeyboardFrameChanged = true
    }
```

Thanks and best regards!! ;)

**Comment by nathantannar4** [comment]
This will be fixed in release 1.1 so stay tuned 😄 1.1 is just pending release. Use the `development` branch for now if you need an immediate fix. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SD10** [comment]
Duplicate of #755 -- or at least similar
